graf3d/eve7/src/REveManager.cxx

```diff
@@ -600,6 +601,25 @@ TGeoManager *REveManager::GetDefaultGeometry()
    return GetGeometryByAlias("Default");
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Get the default viewer.
+///
+
+REveViewer *REveManager::GetDefaultViewer() const
+{
+   return dynamic_cast<REveViewer*>(fViewers->FirstChild());
+}
+
+////////////////////////////////////////////////////////////////////////////////
+/// Enable multiple connections. Disable loopback.
+/// Keep connection key but disable mandatory authentification key.
+///
+void REveManager::AllowMultipleRemoteConnections()
+{
+   ROOT::RWebWindowsManager::SetLoopbackMode(false);
+   fWebWindow->SetRequireAuthKey(false);
```

Comment:
@alja 

Deactivation of authentication should not be here.
It is strong security setting and must appear only in user code for some specific application.

Reply:
@linev 
As a developer it would be convenient to have just one call to enable testing in root macros instead of include of RWebwindowsManager and two function calls.

What if I add a flag to disable AuthKey?
```
void REveManager::AllowRemoteConnections(bool authKey = true)
{
   ROOT::RWebWindowsManager::SetLoopbackMode(false);
   fWebWindow->SetRequireAuthKey(authKey);
}

```

Let me know if you have any other suggestion.

Reply:
@linev I added two non-default arguments to prevent accidental security risk. https://github.com/root-project/root/pull/14912/commits/49566a2dce97ab924233d083be7854d6715f8aac